tsdb/head_append.go

```diff
@@ -1331,6 +1331,9 @@ func (a *headAppender) commitFloats(b *appendBatch, acc *appenderCommitContext)
 					H:   &histogram.Histogram{Sum: s.V},
 				})
 				b.histogramSeries = append(b.histogramSeries, series)
+				// This sample was counted as a float but is now a histogram
```

Comment:
Nit: End the sentence with a period.

```suggestion
				// This sample was counted as a float but is now a histogram.
```